bibliopixel/gamma.py

```diff
@@ -17,7 +17,7 @@ def get(self, i):
 # From https://github.com/scottjgibson/PixelPi/blob/master/pixelpi.py
 APA102 = Gamma(gamma=2.5, offset=0.5)
 WS2801 = SM16716 = Gamma(gamma=2.5)
-DEFAULT = Gamma()
+NONE = DEFAULT = Gamma()  # For when you need no correction
```

Comment:
Unneeded choice doesn't make people happy, you know!  :-)  But sure, not a biggie.